### PR TITLE
change page grouping according to #81

### PR DESCRIPTION
**Pull request by kba**
Must be by explicit physical structMap and file pointers, not implicitly via GROUPID

### CODE REVIEW

File: mets.md [hunk 1]
```diff
@@ -159,7 +170,7 @@ Always use URL. If it's a local file, prefix absolute path with `file://`.
 
 ```xml
 <mets:fileGrp USE="OCR-D-SEG-BLOCK">
-    <mets:file ID="OCR-D-SEG-PAGE_0001" GROUPID="OCR-D-IMG_0001" MIMETYPE="application/vnd.prima.page+xml">
+    <mets:file ID="OCR-D-SEG-PAGE_0001" MIMETYPE="application/vnd.prima.page+xml">
```
Comment (VolkerHartmann) [inline mets.md]: Did we overlook that? Doesn't it have to be OCR-D-SEG-BLOCK_0001 instead of OCR-D-SEG-PAGE_0001 as file@ID?

File: mets.md [hunk 2]
```diff
@@ -159,7 +170,7 @@ Always use URL. If it's a local file, prefix absolute path with `file://`.
...
         <mets:FLocat xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="file:///path/to/workingDir/segmentation/block/page_0001.xml" />
```
Comment (VolkerHartmann) [inline mets.md]: Shouldn't the subdir be SEG-BLOCK or SEG/BLOCK? But in the end it doesn't matter.

File: cli.md [hunk 3]
```diff
@@ -34,9 +34,12 @@ File group(s) used as output.
 
 ## Optional parameters
 
-### `-g, --group-id GROUPID`
+### `-g, --group-id ID`
 
-The `mets:file@GROUPID` to process. If no groupID is defined all files within the `inputGrp` will be processed. Repeatable and multiple IDs can be provided separated by comma.
+The `ID` of the `mets:div[@TYPE='page']` that contains the pointers to files
```
Comment (VolkerHartmann) [inline cli.md]: Alternatively: The `mets:div[@TYPE='page']@ID` that

File: cli.md [hunk 4]
```diff
@@ -34,9 +34,12 @@ File group(s) used as output.
...
+representing a page. Effectively, only those files in the [input
+group](#-i---input-file-grp-grp) that are referenced in these
```
Comment (VolkerHartmann) [inline cli.md]: Suggestion: Effectively, only those files in the [input file group] (#....)

File: cli.md [hunk 5]
```diff
@@ -34,9 +34,12 @@ File group(s) used as output.
 
 ## Optional parameters
 
-### `-g, --group-id GROUPID`
+### `-g, --group-id ID`
```
Comment (VolkerHartmann) [inline cli.md]: Suggestion: -g, --page-id ID
group-id is only understandable if you have the old definition in mind.
Unfortunately -p is already defined.

File: mets.md (hunk 1 above)
Comment (kba) [reply mets.md]: I think you're right.

File: mets.md (hunk 2 above)
Comment (kba) [reply mets.md]: Yeah, the point here was to have an arbirtrary file URL I think.

File: cli.md (hunk 3 above)
Comment (kba) [reply cli.md]: If so, then `mets:div[@TYPE='page']/@ID` so it's a proper XPath. No preference as long as it's consistent.

File: cli.md (hunk 4 above)
Comment (kba) [reply cli.md]: :+1:

File: cli.md (hunk 5 above)
Comment (kba) [reply cli.md]: It is clearer true, but we'd have to change a lot of code for it.

**Pull request merged**